### ISSUE TITLE
Recently played track tile taller than artist and album tiles

### ISSUE THREAD

**Issue by OzGav** [music-assistant/hass-music-assistant#1980]
### What version of Music Assistant has the issue?

2.0.0b106

### What version of the Home Assistant Integration have you got installed?

2024.2.0

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

Looking at the home view of MA the recently playerd tracks tiles are noticeably taller that artist and album tiles

![image](https://github.com/music-assistant/hass-music-assistant/assets/19848947/5e80fae2-01d8-45a9-8d28-1da2b300b81f)

### How to reproduce

Observe home screen

### Music Providers

All

### Player Providers

All

### What version of Home Assistant Core are your running

2024.2.0

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Raspberry Pi

**Comment by OzGav** [comment]
As an option move the icons to the top right corner of the artwork. A dark gray heart with a thin white drop shadow should keep it visible. Then the HIRES logo, if applicable, could show below that?

**Comment by marcelveldt** [comment]
Nah, that would look really crowded. I'm sure we can fix this better with common patterns.

**Comment by marcelveldt** [comment]
I think this one is now fixed (b145) but needs some extra testing
